src/tools/mdbm_config.cc

```diff
@@ -42,7 +42,6 @@ static uint SYS_PAGESIZE;    // system page size
 static uint CONFIG_MAXPAGE = 1 << 24;   // 16MB (subtract system page size later)
 static const int MBYT = 1024 * 1024;
 static const uint ENTRY_OVERHEAD  =   8;     // MDBM per-entry overhead is 8 bytes
-static const uint PAGE_OVERHEAD   =  16;     // MDBM per-page overhead
```

Comment:
I'm not sure that V2 even had the configuration tool...
I think it was the start of an attempt to be more accurate about what fit on a page and when large/oversize pages were needed (It is the correct size for the page header, but you could always use MDBM_PAGE_T_SIZE instead). Note, you could also use MDBM_ENTRY_T_SIZE instead of ENTRY_OVERHEAD, or at least replace the arbitrary-looking '8'.
In practice, the 16 bytes probably didn't matter much for fit calculations. If you need large or oversize pages, you're already pushing Kb-sized or larger objects around.